packages/@webex/plugin-meetings/src/member/util.ts

```diff
@@ -164,11 +164,18 @@ MemberUtil.isAudioMuted = (participant: any) => {
  * @param {Object} participant the locus participant
  * @returns {Boolean}
  */
-MemberUtil.isVideoMuted = (participant: any) => {
+MemberUtil.isVideoMuted = (participant: any): boolean => {
```

Comment:
These changes don't look right to me. The implementation doesn't match `MemberUtil.isAudioMuted`:
If `participant.controls.video.muted === false` we will return `false`, completely ignoring the local mute status, while `isAudioMuted` takes local mute into account.
Also I think the logic for audio and video should be the same, so it would be better for all the logic for local and remote mute to be done inside the common `MemberUtil.isMuted` function and `isVideoMuted` and `isAudioMuted` should just call it with the right parameters for video and audio

Reply:
Yep, you are correct.  Fixed.